home/.zsh/environment.zsh

```diff
@@ -5,7 +5,9 @@ export TERM=xterm-256color
 
 export FZF_CTRL_R_OPTS="--inline-info --exact"
 
-eval "$(dircolors ~/.dircolors)"
+if whence dircolors >/dev/null; then
```

Comment:
I do a similar check for i3 in my .xinitrc file. If that check works on OSX, would you mind updating this so the checking style is consistent? `command -v dircolors > /dev/null && dircolors ~/.dircolors`